### PR TITLE
Add UTs for verifyLockSenderIsWhitelisted in WhitelistSupportImplTest

### PR DESCRIPTION
**Pull request by wilmerrootstock**
## Description
As part of Bridge refactors, this is time to work on the Whitelist process, so we need to extract this from the Bridge and create its objects to decouple the process.

Create UTs for verifyLockSenderIsWhitelisted to increase coverage on WhitelistSupportImplTest.

## Motivation and Context
Increase coverage.

## How Has This Been Tested?
Unit Tests

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Requires Activation Code (Hard Fork)

* **Other information**:

### CODE REVIEW

**Review by apancorb — APPROVED** [review]
LGTM

File: rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java [hunk 1]
```diff
@@ -262,6 +264,33 @@ void setLockWhitelistDisableBlockDelay_whenDisableBlockDelayPlusBtcBlockchainBes
...
     }
 
+    @Test
+    void verifyLockSenderIsWhitelisted_whenOneOffLockWhitelistAddressIsWhitelisted_shouldReturnTrue() {
+        Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, WhitelistCaller.AUTHORIZED.getRskAddress());
+        whitelistSupport.addOneOffLockWhitelistAddress(tx, btcAddress.toString(), BigInteger.TEN);
+
+        boolean actualResult = whitelistSupport.verifyLockSenderIsWhitelisted(btcAddress, Coin.SATOSHI, 0);
+
+        assertTrue(actualResult);
```
Comment (marcos-iov) [inline rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java:274]: I think we should also test that the address is consumed after the verification

Comment (wilmerrootstock) [reply rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java:276]: Done!

Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java:276]: What about in the other test?

Comment (wilmerrootstock) [reply rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java:276]: Unlimited whitelisting means that the entries are never fully consumed so there's nothing to do here.

Comment (marcos-iov) [reply rskj-core/src/test/java/co/rsk/peg/whitelist/WhitelistSupportImplTest.java:276]: Cool, thanks for the clarification

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**